Makefile

```diff
@@ -1,3 +1,10 @@
 DOCKER_REPO             ?= prombench
 
-include Makefile.common
\ No newline at end of file
+.PHONY: all
+all: precheck style check_license lint build test unused
+
+include Makefile.common
+
+.PHONY: unused
+unused:
```

Comment:
Why can't we use the unused one from the commond makefile?

Reply:
in my environment, it will run `GO111MODULE=on go mod tidy` here, clear go.mod and go.sum, then result the next step `git diff --exit-code -- go.sum go.mod` failed.
I think because we don't have any .go file under root.
`go build` will automatically update go.mod / go.sum, it's safe to run `git diff --exit-code -- go.sum go.mod` here.
`go mod tidy` should be run before commit.

Reply:
Oh, found why. It's because of my environment using symbolic link as repository root. `go mod tidy` doesn't follow symbolic link. 
So, I will leave `unused` to `Makefile.common`.